.github/workflows/issues.yml

```diff
@@ -9,9 +9,11 @@ jobs:
 
     steps:
       - name: Git Issue Details
+        env:
+          TITLE: ${{ github.event.issue.title }}
         run: |
           echo "Issue creator: ${{ github.event.issue.user.login }}"
-          echo "Issue title: ${{ github.event.issue.title }}"
+          echo "Issue title: $ISSUE_TITLE"
```

Comment:
@cristiGuranIonos isn't this supposed to be `$TITLE`? I see that above for the `env` variable it is defined as `TITLE`

Reply:
done